### PR TITLE
RC: Document that shards are eventually consistently updated with ACL-s

### PR DESCRIPTION
**Pull request by cmilesb**
Ticket: DOC-2274
Staging link: 
https://redis.io/docs/staging/DOC-2274/operate/rc/security/access-control/data-access-control/active-active-roles/
https://redis.io/docs/staging/DOC-2274/operate/rc/security/access-control/data-access-control/create-roles/

### CODE REVIEW

File: content/integrate/amazon-bedrock/set-up-redis.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ After you store this secret, you can view and copy the [Amazon Resource Name (AR
 
 ## Create a vector index in your database {#create-vector-index}
 
-After your database is set up, create an index with a vector field using [FT.CREATE]({{< relref "/commands" >}}/ft.create/) as your knowledge base for Amazon Bedrock. You can accomplish this using **RedisInsight** or `redis-cli`.
+After your Redis Cloud database is set up, create a search index with a vector field using [FT.CREATE]({{< relref "/commands" >}}/ft.create/) as your knowledge base for Amazon Bedrock. You can accomplish this using **RedisInsight** or `redis-cli`.
```
Comment (dwdougherty) [inline content/integrate/amazon-bedrock/set-up-redis.md:195]:
```suggestion
After your Redis Cloud database is set up, create a search index with a vector field using [FT.CREATE]({{< relref "/commands" >}}/ft.create/) as your knowledge base for Amazon Bedrock. You can accomplish this using **Redis Insight** or `redis-cli`.
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
There will likely be some merge conflicts. I just merged changes to change RedisInsight to Redis Insight. There one fix below for this.

**Comment by cmilesb** [comment]
> There will likely be some merge conflicts. I just merged changes to change RedisInsight to Redis Insight. There one fix below for this.

I just merged main into this branch, so that should eliminate the conflicts.

**Review by dwdougherty — APPROVED** [review]
LGTM

**Pull request merged**